vocabularies/UI.xml

```diff
@@ -1448,6 +1448,11 @@ or `204 No Content` if the request included a `Prefer` header with a value of `r
         </Annotation>
       </Term>
 
+      <Term Name="AdaptationHidden" Type="Core.Tag" Nullable="false" AppliesTo="Property EntitySet EntityType">
+        <Annotation Term="Core.Description" String="Properties or entities annotated with this term can't be used for UI adaptation/configuration/personalization" />
```

Comment:
```suggestion
        <Annotation Term="Core.Description" String="Properties, parameters and entities annotated with this term can't be used for UI adaptation/configuration/personalization" />
```

Reply:
Why are you mentioning parameters there in the "description"? Not sure if this term makes sense for parameters, and if so, we have to add parameters to the AppliesTo attribute as well.